extensions/stats/plugin.h

```diff
@@ -75,10 +74,10 @@ const std::string default_stat_prefix = "istio";
   FIELD_FUNC(destination_canonical_service)  \
   FIELD_FUNC(destination_canonical_revision) \
   FIELD_FUNC(request_protocol)               \
-  FIELD_FUNC(response_code)                  \
-  FIELD_FUNC(grpc_response_status)           \
   FIELD_FUNC(response_flags)                 \
-  FIELD_FUNC(connection_security_policy)
+  FIELD_FUNC(connection_security_policy)     \
```

Comment:
since field ordering here is important, can we add some comments / warning here in case we decide to remove fields or add new ones, etc ?

Reply:
Done.